tests/features/asset_release/add_asset_release.feature

```diff
@@ -104,3 +104,22 @@ Feature: "Add release" visibility options.
     When I am logged in as a moderator
     And I go to "/rdf_entity/http_e_f_fexample1regression/asset_release/add"
     Then I should see the heading "Page not found"
+
+  @javascript
+  Scenario: Check publish time release is present time and past.
+    Given the following owner:
+      | name                     | type    |
+      | Organisation example two | Company |
+    And the following solutions:
+      | title          | description        | documentation | owner                    | state     |
+      | Release Test 2 | test description 2 | text.pdf      | Organisation example two | validated |
+    When I am logged in as a "facilitator" of the "Release Test 2" solution
+    And I go to the homepage of the "Release Test 2" solution
+    And I click "Add release" in the plus button menu
+    Then I should see the heading "Add Release"
+    And I should see the workflow buttons "Save as draft, Publish"
+    And I fill in "Name" with "Release of the dark ship1"
+    And I fill in "Release number" with "1"
+    And I fill in "Authored on" with the date "30-08-2022"
```

Comment:
I see that this step completely doesn't work as expected when using Selenium. The browser is using the user locale from the docker container to format the date field, it uses MM-DD-YYYY. When Mink types in the date, the invalid numbers are dropped and the browser ends up with "03/01/2022" which is accidentally also a date in the future. So this test just happens to work accidentally.

I recall from earlier interactions with the HTML date field that this is NOT controllable by the HTML code. It is probably not even in the HTML5 spec.

I would avoid using the `@javascript` tag for any test unless we are really interacting with JS code that we wrote ourselves. I did not test otherwise.